types/index.d.ts

```diff
@@ -68,6 +68,7 @@ declare module 'opening_hours' {
     sep_one_day_between: string
     zero_pad_month_and_week_numbers: boolean
     locale: string
+    date_format: string
```

Comment:
This will restrict the input to match the used `toLocaleString` API:
```suggestion
    date_format: Intl.DateTimeFormatOptions['weekday']
```
![image](https://github.com/user-attachments/assets/a6380a1f-38f0-40f4-8844-3103ee8172e0)